app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,122 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        create_signing_key: Authentication::AuthnJwt::SigningKey::CreateSigningKeyInterface.new,
+        fetch_issuer_value: Authentication::AuthnJwt::ValidateAndDecode::FetchIssuerValue.new,
+        fetch_identity_from_token: Authentication::AuthnJwt::IdentityProviders::IdentityFromDecodedTokenProvider,
+        validate_webservice_is_whitelisted: ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+        validate_role_can_access_webservice: ::Authentication::Security::ValidateRoleCanAccessWebservice.new,
+        validate_webservice_exists: ::Authentication::Security::ValidateWebserviceExists.new,
+        enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV),
```

Comment:
i think that this is a bug and `enabled_authenticators` needs to be an input. If you initialize this as a dependency then authenticators that are added in runtime (via the db for example) will not be present in the enabled authenticators list. moving `enabled_authenticators`  to the inputs will set the list for each request and thus is correct.

Reply:
You're right, I didnt think about this scenario